pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -158,22 +160,34 @@ func (o *capability) UnregisterFromWorkflow(ctx context.Context, request capabil
 	return nil
 }
 
-func (o *capability) Execute(ctx context.Context, callback chan<- capabilities.CapabilityResponse, request capabilities.CapabilityRequest) error {
+func (o *capability) Execute(ctx context.Context, callback chan<- capabilities.CapabilityResponse, r capabilities.CapabilityRequest) error {
```

Comment:
Fix janky naming, we have `r`, `request`, and `fullRequest` in this context :nauseated_face: